libs/components/src/shared/button-like.abstraction.ts

```diff
@@ -1,4 +1,5 @@
 export abstract class ButtonLikeAbstraction {
   loading: boolean;
   disabled: boolean;
+  setButtonType: (value: "primary" | "secondary" | "danger" | undefined) => void;
```

Comment:
suggestion from @eliykat: don't use undefined
```suggestion
  setButtonType: (value: "primary" | "secondary" | "danger" | "unstyled") => void;
```

Reply:
Fixed!